docs/operations-guide/running-metabase-on-azure.md

```diff
@@ -0,0 +1,134 @@
+- An Azure Database for PostgreSQL server: a database server that Metabase uses as the application database where it saves all its data. This component can be switched to an Azure Database for MySQL if your tune is the MySQL one.
+- A private endpoint connection: so database can be accessed from an internal IP address rather than being exposed to the public.
+- A web app: a deployment of an application fully managed by Azure, either by an executable or a Docker container.
+
+## Steps
+
+### Step 1: Create the resource group
+
+On the Azure management console, click on the **Resource Groups** icon at the top of the page or search for Resource Groups to enter that section. If you have a resource group already created, you can skip this step and go straight to [Step 2](#step-2-create-the-network).
+
+On the resource group page, click on the **+ Add** button in the top bar to create a new resource group. On the **Create a resource group** page that opens, select your valid Azure Subscription, enter a resource group name and select a region.
+
+![Create a Resource Group](images/AZResource_group_Add.png)
+
+Once you've selected the name and region of your resource group click on **Next** on the button at the bottom of the page till you get to the final screen where you can hit the **Create** button.
+
+__NOTE__: remember that you need to select the region for your Metabase instance according to the location of your users, data warehouse and even the cost of the infrastructure of the region or privacy laws that might restrict cross-border data transfers.
+
+### Step 2: Create the network
+
+Once your resource group is created, click on the name to open the page where you will start creating all your resource. Click on the **Create resources** button at the center of the page and you will be taken to the marketplace menu where you will have to search for the keyword **VNET**:
+
+![VNET Marketplace](images/AZMarketPlaceVnet.png)
+
+From all the offerings, select the VNET product from Microsoft and click on **Create**:
+
+![VNET Product](images/AZVnet.png)
+
+You will be presented with a screen similar to the one you were shown when created the resource group, where you will have to enter a name for the virtual network and select a region for it (remember to use the same region for all the components you create, you don't want your data to be travelling worldwide!) and click on **Next: IP Addresses** button at the bottom.
+
+Here we will create a simple but secure network topology, composed of a public subnet (a network that is exposed to the internet), and a private network that will contain your application database which will be inaccesible from the outside.
+
+In the IPv4 address you should have a default value of 10.0.0.0/16 (otherwise add that network space to the box) and then add two subnets, one named public, with the subnet address range of 10.0.1.0/24 and one named private with the subnet range of 10.0.2.0/24.
```

Comment:
```suggestion
In the IPv4 address, you should have a default value of 10.0.0.0/16 (otherwise add that network space to the box). Next, add two subnets:

- one subnet named public, with the subnet address range of 10.0.1.0/24, and
- one subnet named private with the subnet range of 10.0.2.0/24.
```